gnuradio-runtime/python/gnuradio/gru/msgq_runner.py

```diff
@@ -43,12 +43,12 @@
 from __future__ import unicode_literals
 
 from gnuradio import gr
-import gnuradio.gr.gr_threading as _threading
+import threading
```

Comment:
Yeah I re-based this against the other PR that @marcusmueller  had submitted, so that I could run my tests against that.  It should cleanly rebase against master now that that is in.